daffodil-test/src/test/scala/org/apache/daffodil/section00/general/TestBase.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section00.general
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.{ AfterClass, Test }
+
+object TestBase {
+  val testDir = "/org/apache/daffodil/section00/general/"
+  val runner = Runner(testDir, "testBase.tdml")
+
+  @AfterClass def shutDown: Unit = {
+    runner.reset
+  }
+}
+
+/*
+ * This test addresses reported issue DAFFODIL-2210
+ * Unhandled exception with leading/trailing space in restriction base
+ */
+class TestBase {
+
+  import TestBase._
+
+  @Test def test_base1(): Unit = { runner.runOneTest("base1") }
+  @Test def test_base2(): Unit = { runner.runOneTest("base2") }
```

Comment:
Same thing regarding more descriptive test names. They should be enough to describe the test without having to look through the parser test case